python/mxnet/gluon/block.py

```diff
@@ -911,6 +911,8 @@ def __init__(self):
         self._monitor_all = False
         self._backend = None
         self._backend_opts = {}
+        self._final_partitioned = False
+        self._cached_op_args = []
```

Comment:
What’s this for and where is it being set?

Reply:
_final_partitioned is a flag used to ensure that we only partition once, it's set in call cachedOp